addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -374,7 +374,7 @@ export class HtmlField extends Component {
         popover.style.left = leftPosition + 'px';
     }
     async commitChanges({ urgent } = {}) {
-        if (this._isDirty() || urgent) {
+        if (this._isDirty()) {
```

Comment:
Commit message:
> In some circumstances, the call to HtmlField._isDirty returns true
because that method compares the original value with the value of
the wysiwyg that could have changed for multiple technical reasons even
though the user did not made any change.
> To be sure the user has made a change, this commit checks that at least
one historyStep was made.

Reply:
Why did you have to remove the `urgent` case?

Reply:
The urgent removal was a mistake, I've updated the code